test/test.js

```diff
@@ -1090,105 +1094,156 @@ describe('node-fetch', () => {
 		});
 	});
 
-	it('should only use UTF-8 decoding with text()', function() {
-		url = `${base}encoding/euc-jp`;
-		return fetch(url).then(res => {
-			expect(res.status).to.equal(200);
-			return res.text().then(result => {
-				expect(result).to.equal('<?xml version="1.0" encoding="EUC-JP"?><title>\ufffd\ufffd\ufffd\u0738\ufffd</title>');
+	describe('external encoding', () => {
+		const hasEncoding = typeof convert === 'function';
+
+		describe('with optional `encoding`', () => {
+			before(function() {
+				if (!hasEncoding) this.skip();
```

Comment:
I... guess I was using skip for its intended purpose (to test the tests lol) but you're right, I shouldn't have kept it in the commit.